workflows/src/main/resources/templates/fallback.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+</head>
+<body>
+<h1>Did not generate the requested file yet.</h1>
```

Comment:
This is also shown when the workflow just does not have any pages or examples, for example. Kind of unintuitive.

Reply:
there are different fallbacks now